### ISSUE TITLE
The  loss of the agent（cart-pole game） is rising

### ISSUE THREAD

**Issue by zhoudoudou** [keras-rl/keras-rl#116]
when i execute  "python examples/dqn_cartpole.py",The screen will displays the printed information,**but  i find  the loss rise continuously**.at the begin the loss os 0.48...,,when the train is end ,the loss become a very big number.
